docs/api.md

```diff
@@ -101,3 +102,44 @@ This is the route that will return all releases to display inside Nextcloud's ap
 Translated fields are stored inside a translations object. They can have any size, depending on if there is a translation. If a required language is not found, you should fall back to English.
 
 Required versions (minimum and maximum versions) are stored in String fields. If a field is empty, this means that there is no version requirement. Minimum and maximum versions are inclusive, e.g. if your app works only on PHP 5.5 and 5.6, the minimum version will be 5.5.0 while the maximum version will be 5.6.*
+
+## Delete An App
+Only an app owner is allowed to delete an app. The owner is set to the first user that pushes a release to the store.
+
+Deleting an app will also delete all releases which are associated with it.
+
+* **Url**: DELETE /api/v1/apps/{app-id}
+  * **Parameters**:
+    * **app-id**: app's id, lower case ASCII characters and underscores are allowed
+* **Example CURL request**:
+
+      curl -X DELETE http://localhost:8000/api/v1/apps/news
+
+* **Authentication**: Basic
+
+* **Returns**:
+  * **HTTP 204**: If the app was deleted successfully
+  * **HTTP 400**: If the app could not be found
+  * **HTTP 401**: If the user is not authenticated
+  * **HTTP 403**: If the user is not authorized to delete the app
+
+## Delete An App Release
+Only an app owner or co-maintainers are allowed to delete an app release. The owner is set to the first user that pushes a release to the store, co-maintainers can be added.
```

Comment:
This should read "Only app owners or co maintainers are allowed..."

and how about "Owner is the user that pushes the first release of an app to the store,..."